src/net_processing.h

```diff
@@ -84,6 +88,9 @@ class PeerManager : public CValidationInterface, public NetEventsInterface
     /** Get statistics from node state */
     virtual bool GetNodeStateStats(NodeId nodeid, CNodeStateStats& stats) const = 0;
 
+    /** Get stats from peer manager. */
+    virtual PeerManagerStats GetStats() const = 0;
```

Comment:
If you're building a function that collects this info, maybe call it `GetInfo()` and combine it with `IgnoresIncomingTxs()` ?

Reply:
Good idea, done!